src/main/resources/messages/MessageBundle_ru.properties

```diff
@@ -62,3 +62,8 @@ settings.general.content.dimension.cap.max.height=Макс Высота (px)
 settings.general.content.dimension.cap.max.width=Макс Ширина (px)
 settings.general.smol.stickers.title=Маленькие Стикеры
 settings.general.smol.stickers.enable=Использовать Маленькие Стикеры
+notification.margin.saved.message=Стикеры будут удерживать этот относительный отступ, в окнах как этот
+notification.margin.saved.title=Позиция Стикера сохранена
+settings.general.sticker.reset.margins=Сбросить отступы Стикера
+settings.general.content.sticker.margin.info=Кликните два раза по стикерам, чтобы сохранить текущую позицию.
+settings.general.content.sticker.ignore.scaling=Ignore Scaling
```

Comment:
@Dragon-0609 

So I found some bugs and ended up making another feature setting, you please translate this for me:

```properties
settings.general.content.sticker.ignore.scaling=Ignore Scaling
```